.github/actions/test-linux/action.yml

```diff
@@ -16,7 +16,9 @@ runs:
         export PDO_MYSQL_TEST_PASS=root
         export PDO_DBLIB_TEST_DSN="dblib:host=127.0.0.1;dbname=master;version=7.0"
         export PDO_DBLIB_TEST_USER="pdo_test"
-        export PDO_DBLIB_TEST_PASS="password"
+        export PHP_OCI8_TEST_USER="system"
+        export PHP_OCI8_TEST_PASS="pass"
+        export PHP_OCI8_TEST_DB="oracle/XE"
```

Comment:
how can I set env also for pdo_oci ext?

Reply:
See https://github.com/php/php-src/blob/b9f4fb8aefaeb3802d6f79f6aad7029b63e488a2/ext/pdo_oci/tests/common.phpt#L15-L18

Reply:
Setting these env vars were not necessary at last?